Makefile

```diff
@@ -20,7 +20,9 @@ DIRS := \
   contrib/broker/k8s \
   contrib/broker/user_provided \
   contrib/broker/server \
-  controller
+  controller \
+	pkg \
```

Comment:
[nit] this doesn't need to be a tab, and it may be good to have lines 19-25 all start with either spaces, or tabs